tests/tests/fv3_ccpp_gfs_v16_RRTMGP_2thrd

```diff
@@ -0,0 +1,84 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v16 w/ RRTMGP test using two openMP threads
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 CCPP GFS v16 w/ RRTMGP using two threads results with previous trunk version"
+
+export CNTL_DIR=fv3_gfs_v16_RRTMGP_2thrd
```

Comment:
If this test has the the same baseline outputs as `fv3_gfs_v16_RRTMGP` please change `CNTL_DIR` to `fv3_gfs_v16_RRTMGP`.

Reply:
Will do